agent/php_agent.h

```diff
@@ -670,6 +672,13 @@ nr_php_op_array_file_name(const zend_op_array* op_array) {
 #endif
 }
 
+#if ZEND_MODULE_API_NO >= ZEND_7_0_X_API_NO /* PHP 7.0+ */
```

Comment:
Granted it may not be inlined like this?

Reply:
The reason why `nr_php_op_array_*_length` were added was to make the `nr_php_op_array_*` API more complete. There already existed `nr_php_function_name` with nicely paired `nr_php_function_name_length` for zend_function's common field and there existed `nr_php_op_array_function_name` but didn't have `nr_php_op_array_function_name_length`. So I thought instead of mixing `nr_php_op_array_function_name` and raw access to zend_function's op_array.function_name->len (or through Zend's API) I decided to make `nr_php_op_array_*` API more complete. Because the API became complete it made it possible to simplify the `zf_to_id` code.